compyle/transpiler.py

```diff
@@ -158,10 +157,11 @@ def __init__(self, backend='cython', incl_cluda=True):
             #define max(x, y) fmax((double)(x), (double)(y))
 
             #ifdef __APPLE__
```

Comment:
We could get rid of this ifdef, right?

Reply:
I am not sure if we should just yet as this was rather apple specific?  Will leave it for now?